PowerShell Contributing.md

```diff
@@ -0,0 +1,479 @@
+# Ten ways of contributing to PowerShell OSS
+
+## Why this blog
+
+I found [the documentation](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md) on how to contribute to PowerShell/PowerShell a bit daunting.
```

Comment:
"PowerShell/PowerShell"I don't think that's what you meant to write.

Reply:
It's a reference to the github.com/powershell/powershell repository, but I realise now, that is mayby not clear enough. If I change it to 'the PowerShell\PowerShell repository`, would that make things better?